packages/console/src/components/ModalLayout/index.module.scss

```diff
@@ -4,19 +4,35 @@
 .container {
   display: flex;
   flex-direction: column;
-  max-width: dim.$modal-layout-max-width;
-  padding: _.unit(8);
+  width: calc(100% - _.unit(12));
```

Comment:
Try `border-sizing: border-box`?

Reply:
the margin collapse, `box-sizing: border-box` won't work in this case.